src/backend/api/reviews.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//GET ---/api/reviews
+router.get("/", async (req, res) => {
+    try {
+      const result = await knex
+      .select("*")
+      .from("review");
+      if (result) {
+        res
+        .status(200).json({ data: result, message: "the review of all meals" });
+      }
+    
+    } catch (error) {
+      console.error("Server Error", error);
+      res.status(500).json({ error: "unable to import data" });
+    }
+  });
+
+  //GET ---/api/meals/:meal_id/reviews
+router.get("/:meal_id", async (req, res) => {
+    const mealId = req.params.meal_id;
+    try {
+      const result = await knex
+      .from("reviews")
+      .where({ meal_id: mealId });
+      if (result) {
+      res
+      .status(200).json({ data: result, message: "The specific meal" });
+      }
```

Comment:
What if the specific meal with this ID does not exist? A status code of `404` would be helpful to the user. 👆